### ISSUE TITLE
Recaptcha unable to load and marking submissions as Spam.

### ISSUE THREAD

**Issue by bryantAXS** [verbb/formie#333]
**Description**
I was seeing many submissions for our form being marked as spam, and after looking in the formie log files, it mentioned the ReCaptcha (v3) was failing. Looking at the console on the front-end I'm seeing this notice: `Unable to find ReCAPTCHA placeholder for #formie-form-795060197533efa53` and I'm assuming those are related.

**Steps to reproduce**
1. Enable ReCaptcha and add to Form
2. Display form on front-end of site.

### Form settings
Please pick from the options below.

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

**Additional info**
- Plugin version: 1.3.14
- Craft version: 3.5.19.1
- Multi-site: No

**Additional context**

We're rendering the form out with `craft.formie.renderForm(slideForm)`. The only somewhat wonky thing is that the form itself is inside a popup, and isn't visible on page load.

**Comment by bryantAXS** [comment]
I also just tried with a ReCaptcha V2 (Invisible) and got the same error.

**Comment by bryantAXS** [comment]
Also, looking at the `form` tag ID, the Notice is saying `Unable to find ReCAPTCHA placeholder for #formie-form-7950601970611852e` but the form tag itself has an ID of `formie-form-7950601970611852e-p-4226`

Maybe that's part of the problem..?

**Comment by engram-design** [comment]
So the `<form>` element shouldn't be getting that ID, the suffix `-p-4226` is for the page container, see my screenshot https://d.pr/i/pEWONi - can you confirm this?

Can you comment on how you're showing the form (Vue, React, jQuery, JS, etc)? There indeed might be some issues with it being hidden, but it depends.

**Comment by engram-design** [comment]
I think this should be fixed. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.16.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by bryantAXS** [comment]
Ahh, ok you're right, that was for the page container.

Essentially it's just a div that's hidden offscreen with a parent element that has `display:none;` I'm primarily using jQuery along with a few other vannila'ish libs. No React or View.

If you checkout www.76commercecenter.com, you'll see a widget popup on the lower right that contains the form.

I just updated to the recent release and it looks like it's giving a different error now.

https://share.getcloudapp.com/WnuBwpyN

**Comment by bryantAXS** [comment]
Actually, false alarm. That error was caused from not allowing our development environment domain in the ReCaptcha settings. It appears to be working!

**Comment by engram-design** [comment]
Yeah, that's what I've been testing with.  And good to know!

**Comment by engram-design** [comment]
Should be fixed in [1.3.17](https://github.com/verbb/formie/releases/tag/1.3.17)
